piqe_ocp_lib/tests/resources/test_ocp_machine_management.py

```diff
@@ -150,3 +151,26 @@ def test_get_machine_node_ref(self, get_test_objects):
         node_obj = node_obj.get_a_node(node_name)
         assert node_obj.kind == "Node"
         assert node_obj.metadata.name == node_name
+
+
+@pytest.mark.skip_if_not_provider("BareMetal")
```

Comment:
@btjd Have added this for skipping in quicklab for now. With instance from Brady then provider is `BareMetal`, so expecting this to work with such instance.

Reply:
Latest Jenkins run is able to skip this successfully.

Reply:
Nice!